### DISCUSSION TITLE
Suggestion: Highlight PBs in statistics section

### DISCUSSION THREAD

**Discussion by mortzion** [cs0x7f/cstimer#355] [Ideas]
Hightlight the averages/means that are session best using the already existing option **Highlight PB's**

If this idea is approved I can create a PR.

Example: 
![image](https://github.com/cs0x7f/cstimer/assets/16354025/06263399-12b0-4c56-a444-5d8b50c2c61b)

**Comment by cs0x7f** [comment]
All values in the best column must be PBs, which will cause the column to be fully highlighted, which looks strange.

**Reply by afedotov** [reply]
IIUC, highlight only if current value is a best value?

**Reply by mortzion** [reply]
> IIUC, highlight only if current value is a best value?

That's correct.

What I want to improve is to be able to quickly glance at the statistics session and check what averages are currently session best. That means only the "row" (e.g. the ao500 and ao2000 in the OP's image) would need a highlight. If the entire row is highlighted or if only one of the values is highlighted (the current column or the best column) is up to debate.

Currently you can know if you broke a session best by checking the scrolling text in the logo, but it's very easy to miss it and it takes a while to scroll through it if it has a lot of text.
